scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# Helper scripts for development
+
+This directory contains several scripts and docker-compose files that help with development of Tobira.
+None of these files should be used in production.
+
+- `auth-proxy`: an nginx implementing an auth proxy as described in `/docs/auth/all-tobira.md`.
+  Listens on port 3090 and forwards requests to 3080, or for login requests to 3091.
```

Comment:
```suggestion
  Forwards requests to port 3090 to port 3080, except for login requests which go to 3091.
```

:shrug:

Reply:
"to port .. to port" sounds a bit meh. But my original also sounds weird. Will figure something out.